data/human/culture syndicate.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+#############################################
+#      Syndicate with Republic Pairing      #
+#############################################
+
+mission "ZZCU Sad Gamblers -S"
+	minor
+	source
+		government "Syndicate"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Passing through the spaceport you see a sizable group of people waiting on a lottery draw. You hear a whoop of success and your attention is drawn as one of them wins a moderate sum in an 12th division win (a day or two worth of standard pay).`
+			`	It's difficult not to pity them as they immediately buy more tickets with the whole value.`
+				decline
+
+#############################################
+#              Syndicate Only               #
+#############################################
+
+mission "ZZCU Abandoned for Disabilities"
+	minor
+	source
+		government "Syndicate"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`You take a shortcut down an alley and are shocked to find several people with wheelchairs or crutches lving in a shanty town along a hidden service road. It seems Syndicate worlds don't have much in the way of welfare systems. (TBD)`
```

Comment:
I'm assuming TBD means this requires further work?

Reply:
Yes - it's just my cliffnotes version. Feel free to put forward a more complete text suggestion, or it'll likely get removed and revisited in a future PR.